gatling-jms/src/main/scala/io/gatling/jms/check/JmsBodyStringCheckMaterializer.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2011-2019 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gatling.jms.check
+
+import io.gatling.core.check.{ CheckMaterializer, Preparer }
+import io.gatling.core.check.string.BodyStringCheckType
+import io.gatling.jms.JmsCheck
+import javax.jms.Message
+
+object JmsBodyStringCheckMaterializer {
+  def apply(): JmsBodyStringCheckMaterializer = new JmsBodyStringCheckMaterializer()
+}
+class JmsBodyStringCheckMaterializer extends CheckMaterializer[BodyStringCheckType, JmsCheck, Message, String](identity) {
```

Comment:
no need for a class AND a companion, the object suffices

Reply:
Ok